Source/WebKit/UIProcess/gtk/KeyBindingTranslator.cpp

```diff
@@ -82,25 +82,22 @@ static void showHelpCallback(GtkWidget* widget, KeyBindingTranslator*)
 }
 #endif
 
-static const char* const gtkDeleteCommands[][2] = {
-    { "DeleteBackward",               "DeleteForward"                        }, // Characters
-    { "DeleteWordBackward",           "DeleteWordForward"                    }, // Word ends
-    { "DeleteWordBackward",           "DeleteWordForward"                    }, // Words
-    { "DeleteToBeginningOfLine",      "DeleteToEndOfLine"                    }, // Lines
-    { "DeleteToBeginningOfLine",      "DeleteToEndOfLine"                    }, // Line ends
-    { "DeleteToBeginningOfParagraph", "DeleteToEndOfParagraph"               }, // Paragraph ends
-    { "DeleteToBeginningOfParagraph", "DeleteToEndOfParagraph"               }, // Paragraphs
-    { 0,                              0                                      } // Whitespace (M-\ in Emacs)
-};
+static constexpr auto gtkDeleteCommands = std::to_array<std::array<ASCIILiteral, 2>>({
+    { "DeleteBackward"_s,               "DeleteForward"_s          }, // Characters
+    { "DeleteWordBackward"_s,           "DeleteWordForward"_s      }, // Word ends
+    { "DeleteWordBackward"_s,           "DeleteWordForward"_s      }, // Words
+    { "DeleteToBeginningOfLine"_s,      "DeleteToEndOfLine"_s      }, // Lines
+    { "DeleteToBeginningOfLine"_s,      "DeleteToEndOfLine"_s      }, // Line ends
+    { "DeleteToBeginningOfParagraph"_s, "DeleteToEndOfParagraph"_s }, // Paragraph ends
+    { "DeleteToBeginningOfParagraph"_s, "DeleteToEndOfParagraph"_s }, // Paragraphs
+    { 0,                                0                          } // Whitespace (M-\ in Emacs)
```

Comment:
Does nullptr not work here?

```suggestion
    { nullptr,                          nullptr                    } // Whitespace (M-\ in Emacs)
```

I think it should, because you must be using the `std::nullptr_t` constructor. Same for `gtkMoveCommands` below.

Reply:
Yes, this works as expected. Will change those before landing.